src/ui/pages/Menu/components/Settings/components/RecoverySeedPhrase/RecoverySeedPhrase.scss

```diff
@@ -0,0 +1,107 @@
+.recovery-page-container {
+  margin-bottom: 0 !important;
+
+  .user-tips {
+    margin-top: 0.75rem;
+    margin-bottom: 1.75rem;
+
+    p {
+      margin: 0 0 0.5rem;
+    }
+
+    .card-details-info-block-inner {
+      background: rgba(187, 187, 187, 0.5);
```

Comment:
We already have a color for this which you can use instead of hardcoding numerical values. You can change this to: `rgba(var(--ion-color-dark-grey-rgb), 0.5)`.